docs/basics/userguide/plot_stats.md

```diff
@@ -0,0 +1,7999 @@
+(plot_stats_page)=
+
+# Plotting and Statistics
+
+Similary to an RL research project or coursework, we would like to compare, the performances of PPO and A2C from [stable-baselines3](stable-baselines3.readthedocs.io/en/master/) on CartPole-v1. As well as draw statistical significant conclusions on which agent is better in terms of average evaluation performances.
+
+## Reminder on agent training with ExperimentManager
+
+We first start by training the agents on 10 seeds each using [ExperimentManagers](https://rlberry-py.github.io/rlberry/generated/rlberry.manager.ExperimentManager.html#rlberry.manager.ExperimentManager) and [StableBaselinesAgent](https://rlberry-py.github.io/rlberry/generated/rlberry.agents.stable_baselines.StableBaselinesAgent.html#rlberry.agents.stable_baselines.StableBaselinesAgent).
+
+
+```python
+from rlberry.envs import gym_make
+from rlberry.agents.stable_baselines import StableBaselinesAgent
+from stable_baselines3 import A2C, PPO
+from rlberry.manager import ExperimentManager
+
+env_id = "CartPole-v1"  # Id of the environment
+
+env_ctor = gym_make  # constructor for the env
+env_kwargs = dict(id=env_id)  # give the id of the env inside the kwargs
+
+first_agent = ExperimentManager(
+    StableBaselinesAgent,  # Agent Class
+    init_kwargs=dict(algo_cls=A2C),
+    train_env=(env_ctor, env_kwargs),  # Environment as Tuple(constructor,kwargs)
+    seed=42,
+    fit_budget=int(1e4),
+    n_fit=10,
+    agent_name="Sb3-A2C",
+)
+second_agent = ExperimentManager(
+    StableBaselinesAgent,  # Agent Class
+    init_kwargs=dict(algo_cls=PPO),
+    train_env=(env_ctor, env_kwargs),  # Environment as Tuple(constructor,kwargs)
+    seed=42,
+    fit_budget=int(1e4),
+    n_fit=10,
+    agent_name="Sb3-PPO",
+)
+first_agent.fit()
+second_agent.fit()
+```
+
+    [38;21m[INFO] 14:38: Running ExperimentManager fit() for Sb3-A2C with n_fit = 10 and max_workers = None. [0m
```

Comment:
Please clean this up. I don't think we need the outputs of experiments in the doc in this way. It can be interesting once to show what to expect but this is clearly too much output.

Reply:
removed all unnecessary outputs.